celements-model/src/main/java/com/celements/model/access/DefaultXWikiDocumentCreator.java

```diff
@@ -49,6 +54,10 @@ public XWikiDocument createWithoutDefaults(DocumentReference docRef) {
   public XWikiDocument create(DocumentReference docRef, String lang) {
     XWikiDocument doc = createWithoutDefaults(docRef, lang);
     doc.setDefaultLanguage(getDefaultLangForCreatingDoc(docRef));
+    if (doc.getDefaultLanguage().equals(doc.getLanguage())) {
```

Comment:
Check before createWithoutDefaults and directly create the XWikiDocumet with the correct language. Even better: move the check inside createWithoutDefaults.

Reply:
calling now `createWithoutDefaults` with correct language.

didn't move the check inside `createWithoutDefaults` since it's the purpose of this method that defaults like the `getDefaultLangForCreatingDoc` aren't accessed.